### ISSUE TITLE
ffq crashes when run has no data

### ISSUE THREAD

**Issue by sbooeshaghi** [pachterlab/ffq#5]
Example 

`ffq -t GSE GSE75790`

Should handle 

```
INFO Parsing run SRR3729917
ERROR 'NoneType' object has no attribute 'text'
```

Gracefully and should continue parsing the rest of the runs instead of stopping.

**Comment by sbooeshaghi** [comment]
Turns out the SRX was located in a different place for this SRR. Still need to handle the case where there are Nonetypes for the Study/Run/Experiment

**Comment by Lioscro** [comment]
Crash fixed df90a671bc0ad049c0545f7fd4eb533004858011.

**Comment by sbooeshaghi** [comment]
Merged to master https://github.com/pachterlab/ffq/pull/10
